### ISSUE TITLE
Game crashes for unknown reasons (maybe related with dead patient)

### ISSUE THREAD

**Issue by 8mikelx8** [CorsixTH/CorsixTH#651]
The crash happens a few seconds after loading the attached saved game.

  * Saved Game : https://drive.google.com/file/d/0B3-GqVib0easWjExWWlaaEVHRzQ/view?usp=sharing

System: Ubuntu 14.04 64 bits
CorsixTH version: v0.40 , self-compiled with default options.

**Comment by wolfy1339** [comment]
Can you post some your gamelog.txt?

**Comment by MarkL1961** [comment]
It didn't crash for me.  I have a feeling this is a repeat of the issue with hell death sound effects as the grim reaper is at work at the start of this save.

**Comment by MarkL1961** [comment]
I can confirm that when I change my sound effects file to French the game will crash, which is a duplicate of issue #588 I believe, so has already been fixed in the trunk.
I don't think the issue was limited to the French sound effects alone.

**Comment by castarco** [comment]
I'm @8mikelx8 's brother. I can confirm the game was running the Spanish language . I've tried to execute CorsixTH from the command line and logging its console output using tee, but I've obtained an empty log file.

I don't know if there are more log files generated automatically by CorsixTH in some place.

I don't think it's important, but we're using the original Theme Hospital assets, obtained from the "free" Origin's distribution.

**Comment by pwaller** [comment]
Do the sounds continue to function, or does everything freeze?

I strongly suspect this is fixed by #656 - for me, the game freezes if #656 is not applied, and it does not freeze with it.

**Comment by MarkL1961** [comment]
As already stated, I think this is fixed and the same as issue#588, which is pretty much verified by his brother.
I don't think it is related to #656 as clipping is not the issue. The issue is around hell deaths and one or more of the sounds for the gates to hell!

I think we could close this @TheCycoONE ?

**Comment by pwaller** [comment]
Ah, apologies @MarkL1961.

**Comment by pwaller** [comment]
@MarkL1961 you should be aware, the game appears to crash when the clipping is broken.

**Comment by MarkL1961** [comment]
@pwaller  I am aware of the clipping issue, thanks.  I don't believe that became a problem until after 0.40 though.
Please read my first two comments.   It didn't crash for me until I changed my language file.

**Comment by TheCycoONE** [comment]
It does sound like a duplicate of #588.  I'll close the issue.  @8mikelx8 if this is not the case, you can reopen.

**Comment by pwaller** [comment]
I apologise if I seemed like I came out of nowhere. I was trying to be helpful - specifically went to the effort of searching the bug tracker for crashes which seemed like they might be related to theclip bug and I went to the trouble of testing this particular savegame and it froze/didn't freeze with the patch. Without the settings on French. I accept though that this may not look like this crash with the knowledge that the clip bug was new. Sorry for the noise!
